easybuild/easyconfigs/q/QCG-PilotJob/trove-classifiers-2024.4.10_fix_setup.patch

```diff
@@ -0,0 +1,11 @@
+diff -Nru trove-classifiers-2024.4.10.orig/setup.py trove-classifiers-2024.4.10/setup.py
```

Comment:
I know it's a fairly trivial patch, but please include a comment explaining the purpose.

Reply:
done